### PR TITLE
Adjust to API changes in OpenColorIO v2

### PR DESCRIPTION
**Pull request by lgritz**
Signed-off-by: Larry Gritz <lg@larrygritz.com>

### CODE REVIEW

File: src/libOpenImageIO/color_ocio.cpp [hunk 1]
```diff
@@ -1200,21 +1208,29 @@ ColorConfig::createDisplayTransform(ustring display, ustring view,
...
     if (getImpl()->config_) {
-        OCIO::ConstConfigRcPtr config         = getImpl()->config_;
-        OCIO::DisplayTransformRcPtr transform = OCIO::DisplayTransform::Create();
+        OCIO::ConstConfigRcPtr config = getImpl()->config_;
+#    ifdef OCIO_v2
+        auto transform = OCIO::DisplayViewTransform::Create();
+        transform->setSrc(inputColorSpace.c_str());
+        if (looks.size()) {
+            getImpl()->error(
+                "createDisplayTransform: looks overrides are now allowed in OpenColorIO v2");
```
Comment (fpsunflower) [inline src/libOpenImageIO/color_ocio.cpp:1217]: typo: not allowed?

Comment (lgritz) [reply src/libOpenImageIO/color_ocio.cpp:1217]: yes, thanks

**Comment by lgritz** [comment]
By the way, I'm expecting a CI failure here because the "bleeding edge" test that is building against OCIO master (where this comes into play) also builds against pybind11 master, which just a few days ago introduced a change that breaks our build. I'm trying to fix that independently of this PR. When I get that fixed, I will rebase this PR on top of that fix, at which point I expect the testsuite to pass.

File: src/libOpenImageIO/color_ocio.cpp (hunk 1 above)
Comment (lgritz) [reply src/libOpenImageIO/color_ocio.cpp]: fixed

**Pull request merged**